cmake/sysbuild/fast_pair/hex.cmake

```diff
@@ -1,12 +1,27 @@
 #
-# Copyright (c) 2022-2024 Nordic Semiconductor
+# Copyright (c) 2022-2025 Nordic Semiconductor
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
 function(fast_pair_hex_pm)
   set(fp_partition_name bt_fast_pair)
 
+  set(fp_partition_name_defined $<TARGET_PROPERTY:partition_manager,PM_BT_FAST_PAIR_NAME>)
```

Comment:
`fp_partition_name_from_pm_config`?

Reply:
I will call it fp_partition_pm_config_name